tools/scan-images.sh

```diff
@@ -34,6 +34,11 @@ touch image-scan-output/clean-images.txt image-scan-output/dirty-images.txt
 # generate a csv summary
 for image in $images; do
   filename=$(basename $image | sed 's/:/\./g')
+  imagename=$(echo $filename | cut -d "." -f 1)
+  rm .trivyignore && touch .trivyignore
+  for vulnerability in $(yq -e .$imagename'_allowed_vulnerabilities' $KAYOBE_CONFIG_PATH/trivy.yml); do
```

Comment:
Does this do the right thing if an image does not have a list defined?

Does it work if the image name has dashes in? e.g. nova-compute